src/offset/utc.rs

```diff
@@ -74,18 +76,33 @@ impl Utc {
     /// let offset = FixedOffset::east(5 * 60 * 60).unwrap();
     /// let now_with_offset = Utc::now().with_timezone(&offset);
     /// ```
+    ///
+    /// # Panics
+    ///
+    /// Panics if the system clock is set to a time in the extremely distant past or future, such
+    /// that it is out of the range representable by `DateTime<Utc>`. It is assumed that this
+    /// crate will no longer be in use by that time.
+    ///
+    /// If panic in this situation is not acceptable, use [`SystemTime::now`] instead, like this:
+    ///
+    /// ```
+    /// # use chrono::{DateTime, Utc};
+    /// # use std::time::SystemTime;
+    /// # let _ =
+    /// DateTime::<Utc>::try_from(SystemTime::now())
+    /// # ;
+    /// ```
+    ///
+    /// This will yield a <code>Result&lt;DateTime&lt;Utc&gt;, [OutOfRange]&gt;</code>,
```

Comment:
Per your later comment, I've gone ahead and removed this entirely, but in case you're wondering, I did this ugly HTML thing so that `OutOfRange` would be a link. As far as I know, the only way to make an intra-doc link inside an inline code element is by using HTML like that. It is admittedly ugly in source code, though.